src/ui/flutter_app/lib/painting/pieces_painter.dart

```diff
@@ -177,13 +179,14 @@ class PiecesPainter extends PiecesBasePainter {
           assert(false);
           break;
       }
-    });
+    }
 
     // draw focus and blur position
 
-    if (focusIndex != invalidIndex) {
-      final int row = focusIndex! ~/ 7, column = focusIndex % 7;
+    final int row = focusIndex! ~/ 7;
```

Comment:
I don't know if this change was modified based on lint's tips or if I refactored it yourself. It would be great if subsequent Git commits could be atomic, with one commit solving only one issue

Reply:
For example, it will be relatively easier to resolve bugs, if you separate changes into two different commits, one is a modification by Lint's hints, another is refactoring by yourself.